02-basics.Rmd

```diff
@@ -275,7 +275,7 @@ z_charcter
 
 ## Saving R objects
 
-You can also save individual R objects in the RDS format.
+You can also save individual R objects in the RDS format. The RDS format is the ? data format for R. (? is this the right term?)
```

Comment:
Yes that's the right term. How about the following?

``` 
You can also save individual R objects in the Rds format. The Rds format is R's native data format, meaning that any R object can be saved as an Rds file.
```

Reply:
Perhaps you could reference it to Excel syaing that 'in the same way that .xls is the native Excel data format.  ?

Reply:
good idea

Reply:
Suggest add the wording below or similar:  
This is comparable to Excel where the native data format is xls.